### ISSUE TITLE
Add output for stopping_power_loss in electron_stop

### ISSUE THREAD

**Issue by Jonsnow-willow** [brucefan1983/GPUMD#633]
In the current implementation of electron_stop.cu, the stopping_power_loss is not calculated and outputted. This variable could provide valuable insights into the stopping power loss during the simulation. I suggest adding a mechanism to output this variable by writing it to a file for further analysis

`
stopping_power_loss += stopping_power * sqrt(v2) * timestep;
`
Additionally, the timestep (an adaptive timestep) would be passed as a parameter to the function.

**Comment by Jonsnow-willow** [comment]
The feature has been implemented. See the PR #634
